src/Form/Form.php

```diff
@@ -127,6 +120,18 @@ public function errors(): array
         return $this->errors;
     }
 
+    public static function exceptionField(Throwable $exception, array $props = [])
+    {
+        $props = array_merge($props, [
+            'name'  => $props['name'],
```

Comment:
It's not important, but do you think this line is necessary? 🤔

Reply:
Hahah, good one! I have no idea what I was thinking here. I just removed it.